src/client/client_radio.cpp

```diff
@@ -182,34 +182,48 @@ void ClientRadioConfig::initClientRadio(ClientRadioConfigContext* in_context)
         clStn[i]->setFrequency(SOAPY_SDR_TX, ch, "RF", cfg->radioRfFreq);
         clStn[i]->setFrequency(SOAPY_SDR_TX, ch, "BB", kUseUHD ? 0 : cfg->nco);
 
-        if (!kUseUHD) {
-            if (info["frontend"].find("CBRS") != std::string::npos) {
-                if (cfg->freq > 3e9)
-                    clStn[i]->setGain(SOAPY_SDR_RX, ch, "ATTN", -6); //[-18,0]
-                else if (cfg->freq > 2e9 && cfg->freq < 3e9)
-                    clStn[i]->setGain(SOAPY_SDR_RX, ch, "ATTN", -18); //[-18,0]
-                else
-                    clStn[i]->setGain(SOAPY_SDR_RX, ch, "ATTN", -12); //[-18,0]
-                clStn[i]->setGain(SOAPY_SDR_RX, ch, "LNA2", 17); //[0,17]
-            }
+        // Unified gains for both lime and frontend
+        if (_cfg->single_gain()) {
```

Comment:
Same comment as above.

Reply:
@rdoosty I have changed according to your comments.